hints/eating-bananas.md

```diff
@@ -0,0 +1,47 @@
+<br>
+<details class="hint-accordion">  
+    <summary>Recommended Time & Space Complexity</summary>
+    <p>
+    You should aim for a solution with <code>O(nlogm)</code> time and <code>O(1)</code> space, where <code>n</code> is the size of the input array, and <code>m</code> is the maximum value in the array.
+    </p>
+</details>
+
+<br>
+<details class="hint-accordion">  
+    <summary>Hint 1</summary>
+    <p>
+    Can you determine the upper bound for the possible answer? How much time does it take Koko to eat a pile with <code>x</code> bananas? 
```

Comment:
> Can you determine the upper bound for the possible answer?

Remove "possible". Also, we should mention that `h` is always greater than or equal to the length of piles, since that is the only reason we can determine the upper bound.